src/OpenRealEstate.Core/ISaleDetails.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OpenRealEstate.Core
+{
+    public interface ISaleDetails

```

Comment:
I'm also thinking that we might removed `ISalePricing` and move the sale pricing into this interface.

Scope for another issue, post this IMO (and i'll do it).